src/main/java/sirius/web/http/WebContext.java

```diff
@@ -591,6 +591,18 @@ public Value get(String key) {
         return Value.EMPTY;
     }
 
+    /**
+     * Returns a value or parameter supplied by the request as List of Strings.
+     * <p>
+     * The individual values must be separated by a comma or a pipe character.
+     *
+     * @param key the key used to look for the value
+     * @return a List of Strings representing the provided data.
+     */
+    public List<String> getAsStringList(String key) {
+        return Arrays.stream(get(key).asString().split("[,|]")).filter(Strings::isFilled).toList();
+    }
```

Comment:
Wouldn't something like that fit better into `sirius.kernel.commons.Value`? And maybe use a CSV parser if escaping is supposed to be supported.

Reply:
This implementation seems oddly specific. A WebContext already supports a parameter being set multiple times and all set values can be fetched via `WebContext#getParameters`.